aws/structure.go

```diff
@@ -160,6 +160,32 @@ func expandEcsVolumes(configured []interface{}) ([]*ecs.Volume, error) {
 			if v, ok := config["root_directory"].(string); ok && v != "" {
```

Comment:
BTW, If I leave this as it is, I get 
> Error: ClientException: Root Directory can not be set when using an EFS Access Point.

For me to make it work, I had to change `v != ""` for `v != "/"`. Not sure how/why it works here. 🤔

Reply:
Maybe because you're missing https://github.com/terraform-providers/terraform-provider-aws/commit/95feb45eabcf1ac1ed332bb71181a95dbe8bbf42#diff-cca2f484af572d380d4255e649d57d1dR198 ?? @DrFaust92 what do you think? I'd need to test if removing the default would work...

Reply:
I can confirm it. Either `v != ""` or `v != "/"` and `Default:  "/",`